tests/run/cpython_capi.pyx

```diff
@@ -5,18 +5,36 @@ from cpython cimport mem
 from cpython.pystate cimport PyGILState_Ensure, PyGILState_Release, PyGILState_STATE
 
 
+cdef short _assert_calloc(short * s, int n):
```

Comment:
CI failed, and I think this is the relevant bit:

```python-traceback
======================================================================
ERROR: runTest (__main__.CythonRunTestCase)
compiling (c/cy2) and running cpython_capi
----------------------------------------------------------------------
Traceback (most recent call last):
  File "runtests.py", line 1278, in run
    ext_so_path = self.runCompileTest()
  File "runtests.py", line 962, in runCompileTest
    self.test_directory, self.expect_errors, self.expect_warnings, self.annotate)
  File "runtests.py", line 1202, in compile
    self._match_output(expected_errors, errors, tostderr)
  File "runtests.py", line 1251, in _match_output
    self.assertEqual(None, unexpected)
AssertionError: None != '24:57: starred expression is not allowed here'

----------------------------------------------------------------------
```

I'm guessing that this is not a legitimate way to pass a pointer in Cython. I selected "allow edits from maintainers", so no need to wait for me to fix it if a maintainer wants to just cut and paste that helper function in the right places and be done with it.